docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,463 @@
+
+- An OpenTelemetry trace per deployment
+- DORA metrics
+
+![the basics](/docs/install/assets/install02.png)
+
+Notice though that the metrics and traces have nowhere to go.
+That will be fixed in a subsequent step.
+
+## Step 1: Install Keptn Lifecycle Toolkit
+
+Install the Keptn Lifecycle Toolkit using Helm:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Keptn will need to know where to send OpenTelemetry traces.
+Of course, Jaeger is not yet installed so traces have nowhere to go (yet),
+but creating this configuration now means the system is preconfigured.
+
+Save this file as `collectorconfig.yaml`:
+
+```yaml
+---
+apiVersion: options.keptn.sh/v1alpha1
+kind: KeptnConfig
+metadata:
+  name: keptnconfig-sample
+  namespace: keptn-lifecycle-toolkit-system
+spec:
+  OTelCollectorUrl: 'jaeger-collector.keptn-lifecycle-toolkit-system.svc.cluster.local:4317'
+  keptnAppCreationRequestTimeoutSeconds: 30
+```
+
+Apply the file and restart KLT to pick up the new config:
+
+```shell
+kubectl apply -f collectorconfig.yaml
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator --watch
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l component=scheduler
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l component=scheduler --watch
+```
+
+## Create Namespace for Demo Application
+
+Save this file as `namespace.yaml`.
+The annotation means that Keptn Lifecycle Toolkit is active for workloads in this namespace.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: keptndemo
+  annotations:
+    keptn.sh/lifecycle-toolkit: enabled
+```
+
+Create the namespace:
+
+```shell
+kubectl apply -f namespace.yaml
+```
+
+## Deploy Demo Application
+
+It is time to deploy the demo application.
+
+Save this manifest as `app.yaml`:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: keptndemo
+  labels:
+    app.kubernetes.io/name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: nginx
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: keptndemoapp
+        app.kubernetes.io/name: nginx
+        app.kubernetes.io/version: 0.0.1
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
+        ports:
+        - containerPort: 80
+---
+apiVersion: apps/v1
+kind: Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  namespace: keptndemo
+spec:
+  selector:
+    app.kubernetes.io/name: nginx
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 80
+```
+
+Now apply:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+Keptn looks for these 3 labels:
```

Comment:
```suggestion
Keptn looks for these 3 annotations:
```

Reply:
I agree that we need to figure out the labels "vs." annotations thing but right now, it seems that labels is the preferred way. Leaving as-is